api/v1/db/oracledb/games-dao.js

```diff
@@ -11,25 +15,23 @@ const conn = appRoot.require('api/v1/db/oracledb/connection');
  */
 const getGames = async (queries) => {
   // parse passed in parameters and construct query
-  const sqlParams = {
-    scoreMin: queries.scoreMin,
-    scoreMax: queries.scoreMax,
-  };
-  if (queries.name) {
-    sqlParams.name = queries.name;
-  }
-  if (queries.developerId) {
-    sqlParams.developerId = queries.developerId;
-  }
+  const sqlParams = {};
+  // iterate through parameters and add parameters in request to the sql query
+  _.forEach(getParameters, (key) => {
```

Comment:
Why getting parameters from the openapi? Shouldn't they be retrieved from the queries?